spdm.rflx

```diff
@@ -1129,28 +1032,144 @@ package SPDM is
+               if Code = Get_Certificate
+            then Challenge_Request
+               if Code = Challenge
+            then Respond_If_Ready_Request
+               if Code = Respond_If_Ready
+            then Key_Exchange_Request
+               if Code = Key_Exchange and Minor_Version = 1
+            then Finish_Request
+               if Code = Finish and Minor_Version = 1
+            then End_Session_Request
+               if Code = End_Session and Minor_Version = 1;
+         Get_Version_Request : Get_Version_Request
+            then null;
+         Get_Capabilities_Request_1_0 : Get_Capabilities_Request_1_0
+            then null;
+         Get_Capabilities_Request_1_1 : Get_Capabilities_Request_1_1
+            then null;
+         Negotiate_Algorithms_Request_1_0 : Negotiate_Algorithms_Request_1_0
+            then null;
+         Negotiate_Algorithms_Request_1_1 : Negotiate_Algorithms_Request_1_1
+            then null;
+         Get_Digests_Request : Get_Digests_Request
+            then null;
+         Get_Certificate_Request : Get_Certificate_Request
+            then null;
+         Challenge_Request : Challenge_Request
+            then null;
+         Respond_If_Ready_Request : Respond_If_Ready_Request
+            then null;
+         Key_Exchange_Request : Key_Exchange_Request
+            then null;
+         Finish_Request : Finish_Request
+            then null;
+         End_Session_Request : End_Session_Request;
+      end message;
+
+   type Response is
+      message
+         Major_Version : Major_Version
+            then Minor_Version
+               if Major_Version = 1;
+         Minor_Version : Minor_Version
+            then Code
+               if Minor_Version <= 1;
+         Code : Response_Code
+            then Version_Response
+               if Code = Version
+            then Capabilities_Response_1_0
+               if Code = Capabilities and Minor_Version = 0
+            then Capabilities_Response_1_1
+               if Code = Capabilities and Minor_Version = 1
+            then Algorithms_Response
+               if Code = Algorithms
+            then Digests_Response
+               if Code = Digests
+            then Certificate_Response_1_0
+               if Code = Certificate and Minor_Version = 0
+            then Certificate_Response_1_1
+               if Code = Certificate and Minor_Version = 1
+            then Challenge_Auth_Response_1_0
+               if Code = Challenge_Auth and Minor_Version = 0
+            then Challenge_Auth_Response_1_1
+               if Code = Challenge_Auth and Minor_Version = 1
+            then Error_Response_1_0
+               if Code = Error and Minor_Version = 0
+            then Error_Response_1_1
+               if Code = Error and Minor_Version = 1
+            then Key_Exchange_Response
+               if Code = Key_Exchange_RSP and Minor_Version = 1
+            then Finish_Response
+               if Code = Finish_RSP
```

Comment:
This message also requires SPDM 1.1.

Reply:
Changed in 40b2073d86b53873a66e7160db66b7f90f15b435